package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-body-scroll-lock",
-  "version": "0.8.0",
```

Comment:
c78800645@gmail.com
Credit. report wiped clean.
Waiting on my 720 Trans score. Bump the other too.